evergreen/config.yml

```diff
@@ -1761,12 +1776,25 @@ buildvariants:
   - name: compile_test_coverage
   - name: finalize_coverage_data
 
+- name: ubuntu-no-app-services
```

Comment:
I just moved this up so that it was with the other ubuntu builders rather than at the bottom of the file after the windows builders. There are no other changes to its config.